ephemeral/002-deploy-pods-volumes.md

```diff
@@ -0,0 +1,46 @@
+The next step is to create a MySQL pod that uses ephemeral storage and a MySQL pod that uses persistent storage provisioned by StorageOS.
+
+Create a MySQL with ephemeral storage:
+
+`kubectl create -f mysql-pod1.yaml`{{execute}}
+
+Create PersistentVolumeClaim and a MySQL pod which references the `fast` StorageClass:
+
+`kubectl create -f storageos-pvc.yaml`{{execute}}
+
+`kubectl create -f mysql-pod2.yaml`{{execute}}
+
+You should see a StorageOS persistent volume:
+
+`kubectl describe pv`{{execute}}
+
+> Usually takes ~30 seconds to come up after it's been created.
+
+You can also confirm StorageOS provisioned the volume with the CLI:
+
+`kubectl exec -ti cli -n kube-system -- storageos get volume`{{execute}}
+
+The link between the PersistentVolumeClaim and the pod comes from the pod
+referencing the PersistentVolumeClaim in the pod's yaml definition.
+
+Run the command to see the ephemeral mysql pod definition:
```

Comment:
People can `cat` the files if they want. Let try to keep the steps small and concise. If someone wants to check, they can look into it. 

If we want to compare things, then we should explicitly show it. I would create another step that just shows in the scenario that one has a pvc and the other doesn't

Reply:
I left only 2 cats inside and described what the difference is.